source/extensions/transport_sockets/tls/context_config_impl.cc

```diff
@@ -19,6 +21,30 @@ namespace Tls {
 
 namespace {
 
+absl::once_flag fipsOnce;
+bool fipsEnabled;
+
+void initFipsEnabled() {
+  std::ifstream file("/proc/sys/crypto/fips_enabled");
+  if (file.fail()) {
+    std::cout << "FIPS " << (fipsEnabled ? "enabled" : "disabled") << "\n";
```

Comment:
FIPS is always disabled here. What's the content of `fipsEnabled` variable at this point? Better to go back to a static message?

Reply:
True, this was a static message and should be reverted to that